### PR TITLE
[AutoMM] Remove PyMuPDF from installation

### PR DESCRIPTION
**Pull request by zhiqiangdon**
*Issue #, if available:*
#3800

*Description of changes:*
Remove PyMuPDF from test installation. Instead, install it in the CI workflow.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |
| onnxruntime==1.17.1 | onnxruntime==1.17.3 |
| onnxruntime==1.17.1 | onnxruntime==1.17.3 |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

File: .github/workflow_scripts/env_setup.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ function install_multimodal {
     python -m pip install --upgrade pytest-xdist
     install_local_packages "multimodal/$1"
     setup_mmcv
-    # python -m pip install --upgrade "mmocr<1.0"  # not compatible with mmcv 2.0
+    python -m pip install "PyMuPDF<=1.21.1"
```
Comment (tonyhoo) [inline .github/workflow_scripts/env_setup.sh:57]: Let's remove this as well, we can disable/remove the doc classification tests

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by tonyhoo** [comment]
Duplicate fix of https://github.com/autogluon/autogluon/pull/4096

**Pull request closed without merging**